SpywareFilter/sections/specific.txt

```diff
@@ -53,6 +50,8 @@
 /analytics.js$domain=freexcafe.com|rusvideos.tv|nakubani.ru|lun.com|songsara.net|newsmax.com|bolgehastanesi.com|ewtc.de|newsdig.tbs.co.jp
 ||assets.adobedtm.com^$domain=mcclatchydc.com|elnuevoherald.com|bellinghamherald.com|sacbee.com|charlotteobserver.com|newsobserver.com|miamiherald.com|ef.ru|algida.com.tr|bmw-motorrad.co.uk|cif.com.tr|clearhaircare.com|domestos.com|dove.com|elidor.com.tr|f1sport.autorevue.cz|geico.com|gittigidiyor.com|hellmanns.com.tr|herkessofraya.com|johnsonsbaby.com.tr|lipton.com|lnk.to|lux.com|magnumicecream.com|neutrogena.com.tr|omo.com|rexona.com|samsung.com|sap.com|schwarzkopf.com.tr|signalturkiye.com|msnbc.com|nbcnews.com|kbb.com|eurosport.ru|timgate.it|intermiles.com|oxfordmail.co.uk
 !
+||cc.naver.com/cc|$image,other,ping
+/^https:\/\/cc\.(loginfra|naver)\.com\/cc\?a=/$image
```

Comment:
Not need to add generic regexp only for two urls. Just create new rules, similar to `||cc.naver.com/cc|$image,other,ping`

Reply:
Okay.

https://github.com/AdguardTeam/AdguardFilters/pull/122182/commits/406cde88caa32b208cb2071efa520718f56d0654